project/src/components/room/review/review-item.tsx

```diff
@@ -0,0 +1,32 @@
+import { Review } from '../../../types/review';
+import { bringFirstCharToUpperCase } from '../../../utils/common';
+import Avatar from '../avatar/avatar';
+import Rating from '../../rating/rating';
+import { bringToSimpleDate, bringToViewDate } from '../../../utils/date';
+
+type ReviewsItemProps = {
+  review: Review;
+}
+
+export default function ReviewsItem({review}: ReviewsItemProps) {
+  return (
+    <li className="reviews__item" key={review.id}>
+      <div className="reviews__user user">
+        <Avatar avatarUrl={review.user.avatarUrl} type='reviews' classNamePrefix='reviews'/>
```

Comment:
В jsx все же более приняты двойные кавычки
```suggestion
        <Avatar avatarUrl={review.user.avatarUrl} type="reviews" classNamePrefix='reviews'/>
```